model/svg_info.go

```diff
@@ -0,0 +1,18 @@
+package model
+
+import (
+	"encoding/xml"
+)
+
+type SVGInfo struct {
+	XMLName xml.Name `xml:"svg"`
+	Width   string   `xml:"width,attr,omitempty"`
+	Height  string   `xml:"height,attr,omitempty"`
+	ViewBox string   `xml:"viewBox,attr,omitempty"`
+}
+
+func GetSVGInfoForBytes(data []byte) *SVGInfo {
+	var svgInfo SVGInfo
+	xml.Unmarshal(data, &svgInfo)
```

Comment:
`xml.Unmarshal` [returns](https://golang.org/pkg/encoding/xml/#Unmarshal) an error, but we're ignoring it here. Propose changing the semantics that if we get a mime-type of `image/svg+xml` but fail to parse it, we log an error but otherwise proceed with the file upload.